### PR TITLE
Fixes to work with collection-2021* conda envs

### PR DESCRIPTION
**Pull request by mrakitin**
These fixed made the startup files to start successfully against the `collection-2020-2.0rc8` conda env. Done with @dmgav during today's deployment visit at HXN.

xref #17 and https://github.com/NSLS-II-HXN/hxnfly/pull/3 and the latest tag of https://github.com/NSLS-II-HXN/hxntools.

# Requires:
- [ ] https://github.com/NSLS-II-HXN/hxnfly/pull/8

### CODE REVIEW

File: profile_collection/startup/00-startup.py [hunk 1]
```diff
@@ -370,7 +372,12 @@ def insert(self, name, doc):
 from nslsii import configure_base, configure_olog
 
 # configure_base(get_ipython().user_ns, db_new, bec=False)
-configure_base(get_ipython().user_ns, db, bec=False)
+# We have an error:
+# UserWarning: Couldn't start log: Log file is already active: 
+# /home/xf03id/logs/ophyd_user_log_xf03id.log
```
Comment (mrakitin) [inline profile_collection/startup/00-startup.py:377]: @jklynch, this is the IPython logging issue we've observed today.

Comment (jklynch) [reply profile_collection/startup/00-startup.py:377]: Maybe it is this line from ipython_ophyd/profile_collection/ipython_config.py:

```python
# Start logging to the given file in append mode.
c.TerminalInteractiveShell.logappend = '/home/xf03id/logs/ophyd_user_log_xf03id.log'
```

Comment (jklynch) [reply profile_collection/startup/00-startup.py:377]: You could specify that file with BLUESKY_IPYTHON_LOG_FILE.

Comment (mrakitin) [reply profile_collection/startup/00-startup.py:377]: Indeed! Not sure why I wasn't able to find it with `git grep`... probably I was in the `startup/` directory that time.

That's the place in the code:
https://github.com/NSLS-II-HXN/ipython_ophyd/blob/d59ecd4799f4d212525aca1e1956cc989405bb48/profile_collection/ipython_config.py#L297-L298

**Comment by dmgav** [comment]
CI fails because `change-xspress3-imports` branch of `hxnfly` was merged.

**Comment by dmgav** [comment]
I manually added changes from https://github.com/NSLS-II-HXN/profile_collection/pull/22 to avoid long rebase.

**Comment by danielballan** [comment]
This fails CI but it is what the beamline is actually running on. Merging.

**Pull request merged**